src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -128,6 +136,27 @@ public void testKeyEncryption() throws InvalidParametersException {
     }
 
     @Test
+    public void testAesdefaultmodepadding() throws InvalidParametersException {
+        String msg = "Test AES Default";
+        String decrptedMsg;
+        byte[] decrypted;
+        decrptedMsg = StringHelper.encrypt(msg);
+
+        try {
+            SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+            Cipher cipher = Cipher.getInstance("AES");
+            cipher.init(Cipher.DECRYPT_MODE, sks);
+            decrypted = cipher.doFinal(StringHelper.hexStringToByteArray(decrptedMsg));
```

Comment:
Ah. If initializing the cipher first before calling getParameters doesn't work, then lets go back to the original method. I'd surprised if there isn't an easy way of obtaining this info though besides testing it on inputs.